src/storyMap/components/StoryMap.js

```diff
@@ -477,7 +477,9 @@ const Scroller = props => {
           filtered.forEach(setLayerOpacity);
         }
       });
-
+    // This is needed for development due to resize observer issue
+    // should not be here on production
+    // scroller.disable();
```

Comment:
question: can you wrap this in a `process.env.NODE_ENV === 'development'` check?